R/FilteredData.R

```diff
@@ -759,6 +801,18 @@ FilteredData <- R6::R6Class( # nolint
             }
           )
 
+          private$filter_panel_ui_id <- session$ns(NULL)
+          observeEvent(
+            eventExpr = input[["filter_turn_onoff"]],
+            handlerExpr = {
+              if (isTRUE(input[["filter_turn_onoff"]])) {
+                self$filter_panel_enable()
+              } else {
+                self$filter_panel_disable()
+              }
```

Comment:
I would suggest to put all of `shinyjs::disable/enable` calls here and use save/restore methods. This would keep `shiny` functionalities nicely decoupled - methods relates only to _data fields_ and `shiny` parts lives inside the modules.
Moreover, I do not see (and I couldn't imagine) other place where we would call `self$filter_panel_enable/disable()`.